### DISCUSSION TITLE
Auto backup not running

### DISCUSSION THREAD

**Discussion by galatea99** [pocketbase/pocketbase#2545] [Q&A] [answered]
Have anyone tried the auto backup on 0.16.0 on Windows?
While manually initialize the backup works perfectly, the autobackup (I set to everyday) didn't fire up at all.
Is it possible to set some kind of indicator on the admin page to see if it's active?
Thanks.

**Comment by ganigeorgiev** [comment]
Are you sure that you've enabled it? What is the used cron expression?

**Reply by ganigeorgiev** [reply]
Just for the test, you can try to set it to something smaller so that you don't have to wait it, like every 2 minutes:

```
*/2 * * * *
```

**Reply by ganigeorgiev** [reply, marked as answer]
I've just tested it locally and it is working fine for me. Eventually you can try the above cron expression and start the executable with `--debug` and see if there is any backup related log in the console.

**Reply by galatea99** [reply]
I use the every day option. and yes, I have enabled auto backup. 
Will try a more frequent cron schedule and see.

**Comment by galatea99** [comment]
OK. 0.16.1 autobackup works.

**Reply by galatea99** [reply]
Just want to clarify Pocketbase uses UTC time and not your local time.

**Reply by ganigeorgiev** [reply]
Note that there are no changes to the backup functionality in v0.16.1 and it should work fine with v0.16.0 too (only the restore operation internals were changed to support container environments).

Also keep in mind that that a daily cron like `0 0 * * *` will work only if the app is available and running at 00:00 (aka. make sure that your server/container is not suspended).

**Reply by ganigeorgiev** [reply]
> Just want to clarify Pocketbase uses UTC time and not your local time.

Yes, we use UTC for the date server operations (_I'll consider mentioning it in the UI_).
